chef/cookbooks/provisioner/templates/default/autoyast-upgrade.xml.erb

```diff
@@ -12,6 +12,7 @@
       <final_reboot config:type="boolean">false</final_reboot>
       <halt config:type="boolean">false</halt>
       <second_stage config:type="boolean">true</second_stage>
+      <ntp_sync_time_before_installation><%= @ntp_servers.first %></ntp_sync_time_before_installation>
```

Comment:
This will possibly fail if there are no ntp servers configured; don't know how autoyast behaves in that case.